docs/concepts/projects.md

```diff
@@ -729,10 +729,54 @@ dependencies = ["flash-attn"]
 no-build-isolation-package = ["flash-attn"]
 ```
 
-You could run the following sequence of commands:
+You could run the following sequence of commands to sync `flash-attn`:
 
 ```console
 $ uv venv
 $ uv pip install torch
 $ uv sync
 ```
+
+Alternatively, you can provide the `flash-attn` metadata upfront via the
+[`metadata-override`](../reference/settings.md#metadata-override) setting, thereby forgoing the need
+to build the package during the dependency resolution phase. For example, to provide the
+`flash-attn` metadata upfront, include the following in your `pyproject.toml`:
```

Comment:
I think the question that comes to mind for me when reading this is, "why would I want to skip building a package?" The answer that comes immediately to mind is "for performance," but I'm not sure that's right. I think this is most useful when building a package in some circumstances is difficult or impossible, and this provides an escape hatch. Maybe that context can be added here? Basically, say more about _why_ users might want to use this feature.